packages/ffe-form-react/src/InputGroup.js

```diff
@@ -127,10 +153,12 @@ InputGroup.propTypes = {
     onTooltipToggle: func,
     /** Use the Tooltip component if you need more flexibility in how the content is rendered. */
     tooltip: oneOfType([bool, node]),
+    hideErrorOnChange: bool,
```

Comment:
Could you please add this new prop to index.d.ts as well?

Reply:
:+1: done